### PR TITLE
C350670 -  Verify radio buttons on the Record types accordion

### PR DESCRIPTION
**Pull request by tomleexx**
[Allure Report] https://jenkins-aws.indexdata.com/job/folioRancher/job/folioTestingTools/job/runCypressTests/4654/allure/

[C350670] https://foliotest.testrail.io/index.php?/cases/view/350670

add new test scripts for C350670

### CODE REVIEW

**Review by nayimovag — APPROVED** [review]

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      });
+    });
+  });
+
+  after('delete test data', () => {
+    Users.deleteViaApi(user.userId);
+  });
+
+  it(
+    'C350670 Verify radio buttons on the Record types accordion (TaaS)',
```
Comment (nayimovag) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js:33]: please add team name as well, like so: 'C350670 Verify radio buttons on the Record types accordion (firebird) (TaaS)'

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import TopMenu from '../../../support/fragments/topMenu';
+import testTypes from '../../../support/dictionary/testTypes';
```
Comment (dnovikau) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js:2]: Please use single line import: 
```JS
import { TestTypes, DevTeams, Permissions } from '../../../support/dictionary';
```

**Review by dnovikau — APPROVED** [review]

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js (hunk 2 above)
Comment (tomleexx) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js]: use single line import

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js (hunk 1 above)
Comment (tomleexx) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js]: C350670 Verify radio buttons on the Record types accordion (Vega) (TaaS)

Comment (nayimovag) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-radio-button.cy.js]: wrong team, please see test case. 
image

**Pull request merged**